bin/trace

```diff
@@ -470,7 +470,7 @@ for $file (@ARGV) {
   $file_args = 0;
 
   # Ignore some files that do not need tracing macros
-  unless ($file eq "H5FDmulti.c" or $file eq "src/H5FDmulti.c" or $file eq "H5FDstdio.c" or $file eq "src/H5FDstdio.c") {
+  unless ($file eq "H5FDmulti.c" or $file eq "src/H5FDmulti.c" or $file eq "H5FDstdio.c" or $file eq "src/H5FDstdio.c" or $file eq "src/H5TS.c") {
```

Comment:
Nothing in H5TS.c has a TRACE macro and this is easier than screwing with bin/trace to get it to stop messing with the file.

Reply:
This is not a good solution.   Those other files are the "public" examples and H5TS.c is not.

Reply:
H5TS has no calls that require the TRACE macro and I have zero interest in debugging the trace script to fix its processing.